dbt/seeds/ccao/schema.yml

```diff
@@ -0,0 +1,35 @@
+seeds:
+  - name: ccao.class_dict
+    description: '{{ doc("seed_class_dict") }}'
+    config:
+      column_types:
+        min_age: double
+        max_age: double
+        min_size: double
+        max_size: double
+
+    columns:
+      - name: major_class_code
+        description: First digit of class code, major class
+      - name: major_class_type
+        description: Human-readable description of the major class
+      - name: assessment_level
+        description: Level of assessment for the property class
+      - name: regression_class
+        description: Boolean indicating whether or not this class isincluded in CAMA regressions
+      - name: modeling_group
+        description: Modeling group used for internal CCAO data selection
+      - name: reporting_group
+        description: Reporting group name used for internal CCAO reportsand aggregate statistics
+      - name: class_code
+        description: Full, 3-digit class code of the property sub-class
+      - name: class_desc
+        description: Human-readable description of the property sub-class
+      - name: min_size
+        description: Integer of minimum size for 200-class property codes
```

Comment:
We'll have to decide if we want to figure out a way to accommodate infinite values, or change these descriptions from `Integer` to `Double`.

Reply:
Go ahead and switch them to double, since it does support inf types